ckanext/iati_generator/actions/iati.py

```diff
@@ -314,12 +320,19 @@ def iati_resources_list(context, data_dict=None):
     data_dict = data_dict or {}
     toolkit.check_access("iati_file_list", context, data_dict)
 
+    namespace_filter = data_dict.get("namespace")
+
```

Comment:
```suggestion
    namespace_filter = data_dict.get("namespace")
    files_by_resource = h.iati_files_by_resource(namespace=namespace_filter)
```

Implement the filter in the `iati_files_by_resource` function
There we have a DB query available for that